src/test_s2c_srv.c

```diff
@@ -320,7 +330,7 @@ ximfd: xmitsfd = accept(testOptions->s2csockfd,
           if ((s2c_childpid = fork()) == 0) {
             /* close(ctlsockfd); */
```

Comment:
Comment is now out of date and no longer serves the enlightening function that it once did.